main/systemd.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+    "fmt"
+    "os"
+    "os/exec"
+    "strings"
+    "path/filepath"
+)
+
+func contains(distros []string, distro string) bool {
+    if distro != "" {
+        for _, v := range distros {
+            if v == distro {
+                return true
+            }
+        }
+    }
+    return false
+}
+
+func get_distro() string {
+    path, _ := os.Getwd()
+    output, err := exec.Command("sudo", filepath.Join(path, "scripts/distro_script.sh")).Output()
+    if err != nil {
+        errorMessage := fmt.Sprintf("Error getting distro name: %+v", err)
+		errorLogger.Println(errorMessage)
+    }
+    distroName := strings.ToLower(string(output))
+    infoLogger.Println(fmt.Sprintf("distro name: %s", distroName))
+    return distroName
+}
+
+func getSystemdUnitFileInstallPath() string {
```

Comment:
NIT: Since this function is being used outside this module, shouldn't this start with a capital letter? 
```suggestion
func GetSystemdUnitFileInstallPath() string {
```

Reply:
`If the functions with names that start with an uppercase letter will be exported to other packages. If the function name starts with a lowercase letter, it won't be exported to other packages, but you can call this function within the same package.`

Our case this function is being used in same package

Reply:
I mean for now yes, but since its a utility function it can be imported too dont you think?